backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/umsatzreport_adr_bpartner/report_de_CH.properties

```diff
@@ -1,20 +1,33 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
 
-headline=ADR Auswertung
+all = Alle
 
-yes=Ja
-no=Nein
-page=Seite
-of=von
-all=Alle
+attributes = Merkmal
 
-bpartner=Gesch�ftspartner
-issotrx=Art
-periodrange=Zeitperiode
-attributes=Merkmal
+bpartner = Gesch\u00E4ftspartner
```

Comment:
the encoding is not ok; pls do not use eclipse/visual/jreport editor. Use notepad++ or simple notepad and make sure you are not changing the econding.

Reply:
tbh I didn't touch the bpartner portion at all... But I will check on this... I only added org portion but yes I did use the jaspersoft editor for that task... I will also check in the other languages...